roles/setup_gitea/templates/role_binding_sa_to_scc_anyuid.j2

```diff
@@ -0,0 +1,12 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: sa-to-scc-anyuid
+  namespace: "{{ sg_namespace }}"
+subjects:
+  - kind: ServiceAccount
+    name: anyuid
+roleRef:
+  kind: Role
+  name: scc-anyuid
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
Missed line break at the end of the file

Reply:
Done